libs/core/config/include/hpx/config/lambda_capture_this.hpp

```diff
@@ -0,0 +1,16 @@
+//  Copyright (c) 2021 Srinivas Yadav
+//
+//  SPDX-License-Identifier: BSL-1.0
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#pragma once
+
+#include <hpx/config/defines.hpp>
+
+#if defined(HPX_HAVE_CXX20_STD_LAMBDA_CAPTURE)
+#define HPX_PP_IDENTITY(...) __VA_ARGS__
```

Comment:
I know it's a trivial macro, however - could we have the definition of `HPX_PP_IDENTITY()` in a separate header file in the preprocessor module?

Reply:
sure, i will change it soon.